app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    split_integer_result = split_integer(17, 4)
```

Comment:
No need to have this variable
Directly compare `sum(split_integer(17, 4)) == 17`

Fix in places below